prow/deploy.sh

```diff
@@ -69,5 +83,9 @@ trap restore-context EXIT
 ensure-context k8s-prow-builds us-central1-f prow
 ensure-context k8s-prow us-central1-f prow
 echo " $(color-green done), Deploying prow..."
+for s in {5..1}; do
+    echo -n $'\r'"in $s..."
+    sleep 1s
+done
```

Comment:
This is neat. If you add `$'\a'` the count down will beep too 😀

Reply:
don't do that, on many machines it uses a buzzer which is super annoying :stuck_out_tongue:

Reply:
will make sure to start adding $'\a' everywhere

Reply:
testing